app/controllers/immunisation_imports_controller.rb

```diff
@@ -3,8 +3,14 @@
 class ImmunisationImportsController < ApplicationController
   before_action :set_campaign
 
+  def index
+    @immunisation_imports = @campaign.immunisation_imports.order(:created_at)
+  end
+
   def new
     @immunisation_import = ImmunisationImport.new
+
+    render layout: "two_thirds"
```

Comment:
`layout :two_thirds, only: %i[new]` is the more common pattern I think in the rest of our code

Fine with this though

Reply:
I'll update this to match the existing code.